dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedIntervalIterable.java

```diff
@@ -95,11 +162,12 @@ public static ZonedIntervalIterable forConnectorArguments(@Nonnull ConnectorArgu
                 + arguments.getQueryLogEndOrDefault());
 
       LOG.info(
-          "Log entries from {} to {} will be exported in increments of 1 hour.",
+          "Log entries from {} to {} will be exported in increments of {} seconds.",
```

Comment:
Representing in seconds is not very easy to read. I'd suggest to format this in a friendlier way. Commons lang3 has `DurationFormatUtils` for this. See https://www.baeldung.com/java-ms-to-hhmmss for more.